src/scss/form/filter.scss

```diff
@@ -21,3 +21,12 @@
 .filter.active { 
 	.icon.clear { opacity: 1; } 
 }
+.filter.v2 {
```

Comment:
I do not like v2 class name

Reply:
it's temporary solution until transition is done